chainer/cuda.py

```diff
@@ -141,41 +141,51 @@ def __ne__(self, other):
 # ------------------------------------------------------------------------------
 # Global states
 # ------------------------------------------------------------------------------
-def get_device(*args):
-    """Gets the device from a device object, an ID integer or an array object.
 
-    This is a convenient utility to select a correct device if the type of
-    ``arg`` is unknown (i.e., one can use this function on arrays that may be
-    on CPU or GPU). The returned device object supports the context management
-    protocol of Python for the *with* statement.
+def get_device_from_id(device_id):
+    """Gets the device from an ID integer.
 
     Args:
-        args: Values to specify a GPU device. The first device object, integer
-            or :class:`cupy.ndarray` object is used to select a device.
-            If it is a device object, it is returned. If it is an integer,
-            the corresponding device is returned. If it is a CuPy array,
-            the device on which this array reside is returned. If any
-            arguments are neither integers nor CuPy arrays, a dummy device
-            object representing CPU is returned.
+        device_id (int): The ID of the device which this function returns.
```

Comment:
How about `(int or None)` and writing behavior when `None` is given?
And i think this code is enough:
```
if device_id is None:
    return DummyDevice
else:
    return Device(device_id)
```